luskan/README.md

```diff
@@ -0,0 +1,87 @@
+# Storage Inventory TAP service
+
+This service allows queries to the metadata of the Storage Inventory.
+
+## Expected deployment
+This service instance is expected to have:
+- a proxy (HAproxy, apache, nginx) in front that performs SSL termination and forwards calls via HTTP on port 8080. Optional client certificates are passed through 
+via the X-Client-Certificate HTTP header (http-request set-header X-Client-Certificate %[ssl_c_der,base64]
+in haproxcy.cfg).
+- a database backend to store the TAP metadata
+- a reg service to resolve resource ids
+
+
+The container expects that a directory is attached to /conf and containing the following:
+
+## catalina.properties
+System properties required by tomcat:
+
+tomcat.connector.scheme=https
+
+tomcat.connector.proxyName={SSL terminator host name}
+
+tomcat.connector.proxyPort=443
+
+### database connection pool configuration
+luskan.tapadm.maxActive={}
+luskan.tapadm.username={}
+luskan.tapadm.password={}
+luskan.tapadm.url=jdbc:postgresql://postgres/content
+luskan.tapuser.maxActive={}
+luskan.tapuser.username={}
+luskan.tapuser.password={}
+luskan.tapuser.url=jdbc:postgresql://postgres/content
+luskan.uws.maxActive={}
+luskan.uws.username={}
+luskan.uws.password={}
+luskan.uws.url=jdbc:postgresql://postgres/content
+
+### optional intTest configuration
+ca.nrc.cadc.reg.client.RegistryClient.host={SSL terminator host name}
+ca.nrc.cadc.auth.BasicX509TrustManager.trust=true
```

Comment:
remove this trust property
add the property to trust the SSL termination and  X-Client-Certificate header